app/controllers/api/order_cycles_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class OrderCyclesController < BaseController
+    respond_to :json
+
+    def products
+      products = OpenFoodNetwork::ProductsRenderer.new(current_distributor, current_order_cycle, params).products_json
```

Comment:
this is not going to work after #4057 where we merge Api::BaseController and Spree::ApiBaseController (Spree::Core::ControllerHelpers::Order that defines current_order will not be available in the API, which I think is very good).
I dont think we should rely on these global methods. both current_distributor, current_order_cycle they must be taken from params in the controller before being sent to the service.

also, the params variable is something specific to the world of controllers, should never get into the service world. instead of sending params we should get whatever params we need and send them as individual variables to the service. in this case I think that means three separate variables: q, page and per_page